.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,7 +20,7 @@
     - [ ] Updated `README.md` documentation
     - [ ] Updated command list in `/docs/azmcp-commands.md`
     - [ ] Updated test prompts in `/docs/e2eTestPrompts.md`
-    - [ ] For new or modified tool descriptions, ran the `eng/tools/ToolDescriptionEvaluator` tool and obtained a result >= 0.4
+    - [ ] For new or modified tool descriptions, ran the [`ToolDescriptionEvaluator`](https://github.com/Azure/azure-mcp/blob/main/eng/tools/ToolDescriptionEvaluator/Quickstart.md) utility and obtained a score of `0.4` or more and a top 3 ranking for all related test prompts
```

Comment:
```suggestion
    - [ ] For new or modified tool descriptions, run [`ToolDescriptionEvaluator`](https://github.com/Azure/azure-mcp/blob/main/eng/tools/ToolDescriptionEvaluator/Quickstart.md) and obtained a score of `0.4` or more and a top 3 ranking for all related test prompts
```

Reply:
We're talking in the past tense for all other items. I think I'll keep it as such.

Reply:
But will remove the word "utility"